packages/fuel-indexer-schema/src/db/tables.rs

```diff
@@ -419,6 +424,81 @@ impl Schema {
     }
 }
 
+fn get_foreign_keys(schema: &str) -> HashMap<String, HashMap<String, String>> {
+    let mut foreign_keys: HashMap<String, HashMap<String, String>> = HashMap::new();
+
+    let base_ast = match parse_schema::<String>(BASE_SCHEMA) {
+        Ok(ast) => ast,
+        Err(e) => {
+            panic!("Error parsing graphql schema {e:?}",)
+        }
+    };
+    let (primitives, _) = build_schema_objects_set(&base_ast);
+
+    let ast = match parse_schema::<String>(schema) {
```

Comment:
maybe a comment above this line // abstract-syntax-tree

Reply:
I think `ast` is pretty self-explanatory.